cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,95 @@ include::modules/common-attributes.adoc[]
 :context: odo-release-notes
 toc::[]
 
-include::modules/openshift-developer-cli-odo-notable-features.adoc[leveloffset=+1]
+[id="odo-notable-improvements_{context}"]
+== Notable improvements in {odo-title}{nbsp}{odo-ver}
 
-include::modules/openshift-developer-cli-odo-getting-support.adoc[leveloffset=+1]
+* IBM Z and PowerPC architecture binaries are now available.
+* `odo catalog` has been improved to provide more useful output information.
+* Information prompts have been added to the `odo service create` command, providing possible next step scenarios.
+* Log verbosity can now be set with an environment variable `ODO_LOG_LEVEL`.
+* Use `odo preference set PushTimeout <seconds>` to specify the number of seconds `odo` waits for a component pod to be deployed before the command fails.
+* Overall documentation improvements. The documentation now includes a description of {odo-title}'s inner architecture.
```

Comment:
Noticing that odo as a noun is not in back ticks, while oc as a noun in the docs is. Personally, I like it as you have it with no formatting in {odo-title}, but something to consider for consistency. At some point, we should decide on which to use and add to: https://github.com/openshift/openshift-docs/blob/enterprise-4.2/contributing_to_docs/term_glossary.adoc